.travis.yml

```diff
@@ -57,6 +57,7 @@ env:
   - PACKAGE=symbiflow-toolchain-xray
 
 script:
+  - while sleep 9m; do echo "=====[ $SECONDS seconds still running ]====="; done &
```

Comment:
we solved a similar problem in litex-conda-ci: https://github.com/litex-hub/litex-conda-ci/commit/c28fd2676612998875ef4b65ff03277ae8b13583

Please take a look here

Reply:
`trravis_jigger`!

Reply:
Fixed, I implemented similar solution as in litex-conda-ci.